actions/apps.ts

```diff
@@ -3,20 +3,20 @@
 
 import {Client4} from 'mattermost-redux/client';
 import {Action, ActionFunc, DispatchFunc} from 'mattermost-redux/types/actions';
-import {AppCallResponse, AppCall, AppForm} from 'mattermost-redux/types/apps';
-import {AppsBindings, AppCallTypes, AppCallResponseTypes} from 'mattermost-redux/constants/apps';
+import {AppCallResponse, AppCall, AppForm, AppBinding} from 'mattermost-redux/types/apps';
```

Comment:
thanks for removing the `s` from AppsBindings :)

Reply:
It is actually still `AppsBindings`. That's just not being imported in this file anymore. I want to limit the change in this PR so that can be done afterwards